src/Components/Footer/footer.css

```diff
@@ -1 +1,193 @@
-/*# sourceMappingURL=footer.css.map */
\ No newline at end of file
+/*# sourceMappingURL=footer.css.map */
+
+.footer{
+    width: 100%;
+    position: relative;
+    padding: 2rem 0;
+    display: flex;
+    align-items: center;
+    justify-content:center;
+    margin: auto;
+
+     .videoDiv{
+        postion: abosolute;
```

Comment:
there is some minor spelling problem i just mention them and have a look at them